browser.js

```diff
@@ -1,27 +1,7 @@
-var createHmac = require('create-hmac')
-var MAX_ALLOC = Math.pow(2, 30) - 1 // default in iojs
-
-exports.pbkdf2 = pbkdf2
-function pbkdf2 (password, salt, iterations, keylen, digest, callback) {
-  if (typeof digest === 'function') {
-    callback = digest
-    digest = undefined
-  }
-
-  checkParameters(iterations, keylen)
-
-  if (typeof callback !== 'function') {
-    throw new Error('No callback provided to pbkdf2')
-  }
-
-  setTimeout(function () {
-    var result = pbkdf2Sync(password, salt, iterations, keylen, digest)
-
-    callback(null, result)
-  })
-}
+exports._createHmac = require('create-hmac/browser')
```

Comment:
for example [cryptocoinjs/scryptsy](https://github.com/cryptocoinjs/scryptsy)
but if you think that this isn't necessary, we should remove `create-hash/browser` in `create-hmac`